doc/source/tutorial/optimize.rst

```diff
@@ -676,7 +676,7 @@ Both linear and nonlinear constraints are defined as dictionaries with keys ``ty
     ...              'fun' : lambda x: np.array([1 - x[0] - 2*x[1],
     ...                                          1 - x[0]**2 - x[1],
     ...                                          1 - x[0]**2 + x[1]]),
-    ...              'jac' : lambda x: np.array([[1.0, 2.0],
+    ...              'jac' : lambda x: np.array([[-1.0, -2.0],
```

Comment:
because the jacobian was changed then evidently the result also changes on [line 700](https://github.com/scipy/scipy/pull/9131/files#diff-aae85539cc8f886d7326a10fff793127R700) it should now be:
    [0.41494475 0.1701105 ]